### DISCUSSION TITLE
can't get report of gensim LDA model

### DISCUSSION THREAD

**Discussion by franciee2** [maximtrp/tmplot#12] [Q&A]
Hi there! I don't know why I recieved this error with my gensim LDA model, I think I've alreay input 'corpus' variable, but the hints always ask me to supply a corpus. Do you know why this appears? Thanks a lot.
![Screenshot](https://github.com/user-attachments/assets/14425b54-780c-4e88-983a-9eb162f077ae)

**Comment by maximtrp** [comment]
Thank you for reporting! Will look over it

**Comment by maximtrp** [comment]
I have fixed it. Please try the version. And thank you for reporting!
